src/fontra/views/editor/panel-designspace-navigation.js

```diff
@@ -358,21 +359,25 @@ export default class DesignspaceNavigationPanel extends Panel {
   _setupSourceListColumnDescriptions() {
     const columnDescriptions = [
       {
-        title: "on",
+        title: translate("sidebar.designspace-navigation.glyph-sources.on"),
         key: "active",
         cellFactory: makeIconCellFactory(
           ["/tabler-icons/circle-dotted.svg", "/tabler-icons/circle-dot.svg"],
           true
         ),
-        width: "1.2em",
+        width: "2.5em",
```

Comment:
Hm, this changes the layout more than I'd like. Perhaps we need an icon instead of "on" as a label, or leave out the label altogether.